pkg/identityserver/store/api_key_store.go

```diff
@@ -45,6 +45,7 @@ func (s *apiKeyStore) CreateAPIKey(ctx context.Context, entityID ttnpb.Identifie
 		Name:       key.Name,
 		EntityID:   entity.PrimaryKey(),
 		EntityType: entityTypeForID(entityID),
+		ExpiresAt:  key.ExpiresAt,
```

Comment:
How about UpdateAPIKey? Shouldn't users be able to change this after creation?

Reply:
From this comment https://github.com/TheThingsNetwork/lorawan-stack/pull/4052#discussion_r614718052 I got the impression we want it only when creating the key? I don't think we need to allow updating the expiration date when users can just set a new key, but let me know if you meant something else.

Reply:
That CLI command generates an API key for claiming. My comment there was about the fact that passing the expiry flag only makes sense when letting the command generate an API key, not when setting an existing API key.

I think in general users will want to be able to edit the expiry date of an API key.